README.md

```diff
@@ -59,7 +59,7 @@ func main() {
 See the [API
 documentation](https://pkg.go.dev/go.opentelemetry.io/otel) for more
 detail, and the
-[opentelemetry-example-app](./example/README.md)
+[opentelemetry-example-app](./example/)
 for a complete example.
```

Comment:
I think we can rephrase the link title as well. The singular example reference is a hold over that we have long moved beyond.

```suggestion
[opentelemetry examples](./example/).
```